build/gulp/plugins/util/getComponentInfo.ts

```diff
@@ -111,7 +111,29 @@ const getComponentInfo = (filepath: string, checksum?: string) => {
   // sort props
   info.props = _.sortBy(info.props, 'name')
 
+  // available behaviors
+  info.behaviors = getAvailableBehaviors(info.props.accessibility)
+
   return info
 }
 
+const getAvailableBehaviors = accessibilityProp => {
```

Comment:
```suggestion
const getAvailableBehaviors = (accessibilityProp): string[] => {
```

Can we add an input and result type there? It's our internal code, but it we more understandable.

Reply:
added an interface for output, input is part of `any`, so I just added `{ tags: [] }`